shared/src/business/entities/cases/Case.test.js

```diff
@@ -1967,4 +1967,318 @@ describe('Case entity', () => {
       });
     });
   });
+
+  describe('Case Consolidation Eligibility', () => {
+    describe('canConsolidate', () => {
+      let caseEntity;
+
+      beforeEach(() => {
+        caseEntity = new Case(
+          { ...MOCK_CASE, status: 'Submitted' },
+          {
+            applicationContext,
+          },
+        );
+      });
+
+      it('should fail when the pending case status is ineligible', () => {
+        caseEntity.status = 'New';
+        const result = caseEntity.canConsolidate();
+
+        expect(result).toEqual(false);
+      });
+
+      it('should pass when a case has an eligible case status', () => {
+        const result = caseEntity.canConsolidate();
+
+        expect(result).toEqual(true);
+      });
+
+      it('should accept a caseToConsolidate param to check its eligible case status', () => {
```

Comment:
That's the internal name of this argument in the method itself: https://github.com/flexion/ef-cms/blob/develop/shared/src/business/entities/cases/Case.js#L1308

It could probably be more clearly written as: `...a case to consolidate as a param...`
Would you like to me to make a change to that effect?

Reply:
That'd be great — thank you,